pico/tests/resources-pico/src/test/resources/expected/ximpl-interceptor._java_

```diff
@@ -13,7 +13,7 @@ import io.helidon.common.types.DefaultTypeName;
 import io.helidon.common.types.DefaultTypedElementName;
```

Comment:
You need to also do these same prefix <--> suffix changes to `common/types`

Reply:
Pushed